src/main/java/duke/commands/FindCommand.java

```diff
@@ -0,0 +1,58 @@
+package duke.commands;
+
+import java.util.ArrayList;
+
+import duke.exceptions.DukeException;
+import duke.task.Task;
+
+/**
+ * Represents a find command in the Duke application.
+ */
+public class FindCommand extends Command {
+    /** Command word of the find command. */
+    public static final String COMMAND_WORD = "find";
+    private static final String TASK_FORMAT = "%d: %s";
+    private static final String USER_MESSAGE_FORMAT = "Here are the matching tasks in your list!\n%s";
+    private static final DukeException WRONG_FORMAT =
+        new DukeException("Wrong format for Find!\nShould be 'find <keyword>'.");
+    private final String keyword;
```

Comment:
Good use of constants

Reply:
Thanks for taking the time to provide your feedback!